<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterArrivee</class>
 <widget class="QDialog" name="AjouterArrivee">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>509</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>746</width>
    <height>509</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>746</width>
    <height>509</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AjouterArrivee</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Code du vol:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Compagnie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Heure:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ville:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Statut:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>253</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string> Atterri </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>À l’heure</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string> Retardé </string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditVille">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHeure">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCompagnie">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCode">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfirmerArrivee">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirmer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonConfirmerArrivee</sender>
   <signal>clicked()</signal>
   <receiver>AjouterArrivee</receiver>
   <slot>slotAjouterArrivee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAjouterArrivee()</slot>
 </slots>
</ui>
